### PR TITLE
Add `GetExitQueueEpoch`

### PR DESCRIPTION
**Pull request by syjn99**
```plain
GET /over/v1/beacon/states/head/exit/queue_epoch/exit_balance=256000000000
```

Calculate `exit_queue_epoch` for request. Used `InitiateValidatorExit` and `ExitEpochAndUpdateChurn` utils.

### CODE REVIEW

File: beacon-chain/rpc/over/handlers.go [hunk 1]
```diff
@@ -217,6 +220,92 @@ func (s *Server) GetReserves(w http.ResponseWriter, r *http.Request) {
...
+		return
+	}
+	isFinalized := s.FinalizationFetcher.IsFinalized(ctx, blockRoot)
+
+	copiedSt := st.Copy()
+
+	var exitQueueEpoch primitives.Epoch
+	var churn uint64
+	// code is originated from InitiatorValidatorExit (beacon-chain/core/validators/validator.go)
+	if copiedSt.Version() < version.Electra {
```
Comment (sb-anderson) [inline beacon-chain/rpc/over/handlers.go]: Do we have Versions before Electra?

Comment (syjn99) [reply beacon-chain/rpc/over/handlers.go]: This part is added for backward compatibility. Can be removed after cleaning code

File: beacon-chain/rpc/over/handlers.go [hunk 2]
```diff
@@ -288,3 +377,42 @@ func append0x(input string) string {
...
+
+	activeBal, err := helpers.TotalActiveBalance(st)
+	if err != nil {
+		return 0, err
+	}
+	earliestExitEpochFromState, err := st.EarliestExitEpoch()
+	if err != nil {
+		return 0, err
+	}
+	exitBalanceToConsumeFroState, err := st.ExitBalanceToConsume()
```
Comment (sb-anderson) [inline beacon-chain/rpc/over/handlers.go]: typo?

Comment (syjn99) [reply beacon-chain/rpc/over/handlers.go]: Thank you!

File: beacon-chain/rpc/over/handlers.go [hunk 3]
```diff
@@ -101,6 +105,92 @@ func (s *Server) GetReserves(w http.ResponseWriter, r *http.Request) {
...
+	st, err := s.Stater.State(ctx, []byte(stateId))
+	if err != nil {
+		httputil.WriteError(w, handleWrapError(err, "could not retrieve state", http.StatusNotFound))
+		return
+	}
+
+	// Parse exit_balance from URL params
+	rawExitBalance := r.URL.Query().Get("exit_balance")
+	if st.Version() >= version.Alpaca && rawExitBalance == "" {
+		httputil.HandleError(w, "exit_balance is required for post-electra in query params", http.StatusBadRequest)
```
Comment (gazzua) [inline beacon-chain/rpc/over/handlers.go]:
```suggestion
		httputil.HandleError(w, "exit_balance is required for post-alpaca in query params", http.StatusBadRequest)
```

😄

**Review by gazzua — APPROVED** [review]

**Pull request merged**